english/experience/black-point/concentrate/stargaze.md

```diff
@@ -1,6 +1,14 @@
 When you concentrate on the black point for some more time you see the point transform.
 
-The black point transforms back into the 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.